x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/logs/logs_tab_content.tsx

```diff
@@ -5,9 +5,15 @@
  * 2.0.
  */
 
-import React, { useMemo } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
+import { DEFAULT_LOG_VIEW } from '../../../../../../../common/log_views';
+import type {
+  LogIndexReference,
+  LogViewReference,
+} from '../../../../../../../common/log_views/types';
```

Comment:
**note:** just a quick note about these imports, when merging https://github.com/elastic/kibana/pull/159279, this imports will need to be updated since all these definition have been moved into a new plugin.